workflow-job

```diff
@@ -860,6 +861,17 @@ def publish() {
         sh "docker login -u ${dockerUser} -p ${dockerPassword}"
     }
 
+    if (dockerTag) {
+        publishTag(dockerTag)
+    }
+    if (isLatest == "true") {
```

Comment:
as a string?

Reply:
Yes, parameters are converted to Strings while triggering other jobs in JobDSL...